app/views/return-requirements/check.njk

```diff
@@ -103,6 +104,31 @@
         }) }}
       </form>
       {% for requirement in requirements %}
+        {# Generate the points cell #}
+        {# When a requirement has more than 5 points it is displayed as a GOVUK Details component which displays the count #}
+        {# of points as the summary. Users must expand the block to get the full list. #}
+        {% set pointCell %}
+          {% set pointsList %}
+            {% set count = requirement.points | length %}
+            {% for point in requirement.points %}
+              {% set count = loop.index %}
+              {{ point }}<br>
```

Comment:
It's my fault @rvsiyad ! I know you would have copied this from something I did in the bill runs.

I shouldn't have used `<br>` to separate the elements. 🤦

Fortunately, @jonathangoulding added a macro to this view (look at the top of the file) specifically for listing out items (that was the `newLine(requirement.points)` call this change has replaced).

Could you have a crack at incorporating that method rather than reproducing my `<br>` hack?